recipes/cap-mirseq/meta.yaml

```diff
@@ -0,0 +1,35 @@
+about:
+  home: http://bioinformaticstools.mayo.edu/research/cap-mirseq/
+  license: 'GPL'
+  summary: 'A comprehensive analysis pipeline for deep microRNA sequencing'
+
+build:
+  number: 0
+  skip: True # [osx]
+
+package:
+  name: cap-mirseq
+  version: 'version'
+
+requirements:
+  run:
+      - perl-threaded
+      - java-jdk
+      - r-base
+      - r-essentials
+      - bioconductor-edger ==3.14.0
+      - bowtie ==1.1.2
+      - samtools ==0.1.19
+      - picard ==1.97
+      - fastqc ==0.11.5
+      - bedtools ==2.17.0
+      - mirdeep2 ==2.0.0.7
+      - vcftools ==0.1.11
+
+source:
+  fn: cap-mirseq.tar.gz
+  url: http://bioinformaticstools.mayo.edu/research/wp-content/plugins/download.php?url=https://s3-us-west-2.amazonaws.com/mayo-bic-tools/cap-mirna/CAP-miRSEQ.tgz
```

Comment:
Wow, this URL looks scary.  Can you include a checksum? Do you know if this URL is sustainable ... Maybe you can use https://github.com/galaxyproject/cargo-port to make it more stable?
Just a suggestion ....